### PR TITLE
WIP Blog/lwx how design works at supabase

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+published_at: '2023-12-08T09:00:00.000-09:00'
+toc_depth: 3
+author: jonny,marijana,fsansalvadore
+image: lwx-how-design-works-at-supabase/og.jpg
+thumb: lwx-how-design-works-at-supabase/thumb.jpg
+---
+
+After 3 years, only now are we figuring out “what design is” for Supabase and how it functions in the wider org.
+
+We recently became a team of 3, developing a somewhat-unique culture to increase the output and quality of our own team and the product teams. Here are a few insights into what we’ve learned along the way.
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:21]: "We recently became a team of 3" this was deliberate. sounds more impressive.

File: apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+- Aesthetic Principles [add anchor link]
+- Product Design Principles [add anchor link]
+
+### Aesthetic principles
+
+We needed to agree on what we collectively like and dislike on an emotional level. Aligning on a common aesthetic avoids debates on individual elements.
+
+<Img src="/images/blog/lwx-how-design-works-at-supabase/aesthetic-moodboard.png" wide />
+
+Aesthetic mood board of what Design Team like externally
```
Comment (fsansalvadore) [inline apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:86]:
```suggestion
<Img
  src="/images/blog/lwx-how-design-works-at-supabase/aesthetic-moodboard.png"
  wide
  caption="Aesthetic mood board of what Design Team like externally"
/>
```

Comment (fsansalvadore) [reply apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:86]: also has a `caption` prop @MildTomato

File: apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+#### Design for the "Postgres developer"
+
+We are moving towards a SQL first experience in the dashboard. You’ll see some LWX announcements that lean more into empowering developers to learn SQL rather.
+
+#### Consolidation follows kaizen
+
+Product Teams must ship _[kaizen](https://en.wikipedia.org/wiki/Kaizen)_ improvements. But It’s Design Team’s role to consolidate after. This helps with velocity - we are not a blocker in the development process.
+
+For example, yesterday the Frontend team added a “New organization” button in the dashboard based on a message from Kevin (see below). They skipped Design Team input, shipped a quick solution, then the Design team will do a more refined layout update after LWX, improving the page as whole.
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:120]: It was "yesteday" :p

File: apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+### Iterative changes
+
+After we have shipped to prod, design work becomes expired. Whatever is mocked before no longer serves a purpose.
+
+We took some inspiration from [Linear’s approach](https://x.com/karrisaarinen/status/1715091479184736629?s=20) - taking screenshots of what is in prod, and using the screenshots to construct mockups.
+
+Here’s an example where we are making changes to the table editor:
+
+<Img src="/images/blog/lwx-how-design-works-at-supabase/screenshots-in-mockups.png" wide />
```
Comment (fsansalvadore) [inline apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:61]: you could use the `caption` prop on the  component

Comment (fsansalvadore) [reply apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:61]:
```suggestion
<Img src="/images/blog/lwx-how-design-works-at-supabase/screenshots-in-mockups.png" wide caption="A screenshot of the table editor from prod in a mockup" />
```

File: apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+The emphasis is on the daily gains, unblocking problems with design work so that no team is paralyzed by how something should work, function or look. This can manifest in: static mockups, interactive prototypes in Figma, code prototypes, wireframe sketches, and sometimes we actually start building it (👀). We do any work that can help the organization build consensus.
+
+Let’s take an example, our LWX ticket page:
+
+<figure className="[&_img]:border-0">
+  ![something](/images/blog/lwx-how-design-works-at-supabase/lwx-tickets-iterations.png)
+</figure>
+
+Late the in build process an idea was floated to try and integrate a _Wordle_ type easter egg into the site. We build some designs to help the discussion:
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx]: quite like that its present tense. we still do build.

File: apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (RamiroND) [reply apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:21]: Ok cool 👍

File: apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+
+For a long time we operated without any sort of consensus of what Design _is_ at Supabase: what are our values, what do we like, what do we not like?
+
+It’s surprising how far you can get without these. After 3 years, we noticed that we were drifting into contradictory aesthetics and it was becoming challenging to support other teams.
+
+At the founding of Supabase, the team agreed on a set of [Product Principles](https://supabase.com/docs/guides/getting-started/architecture#product-principles). Internally, we’ve expanded this concept to all teams / products / functions at Supabase.
+
+The Design Team adopted and maintain two principle sets:
+
+- Aesthetic Principles [add anchor link]
```
[the remark this answers is not in the document]
Comment (MildTomato) [reply apps/www/_blog/2023-12-08-how-design-works-at-supabase.mdx:77]: fixed

File: apps/www/lib/authors.json [hunk 7]
```diff
@@ -283,8 +283,8 @@
     "author_id": "marijana",
     "author": "Marijana Šimag",
```
Comment (fsansalvadore) [inline apps/www/lib/authors.json:284]:
```suggestion
    "author": "Marijana Pavlinić",
```

Comment (MildTomato) [reply apps/www/lib/authors.json]: ah yea sorry, just changed it

**Review by fsansalvadore — APPROVED** [review]

**Pull request merged**